docs/dashboards/Host Dashboard.md

```diff
@@ -0,0 +1,94 @@
+# OpenTelemetry Host Dashboard
+
+> [!WARNING]
+> The dashboards shared in this repository are in an alpha state and can change significantly.
+> They are provided as-is, with no support guarantees.
+> Newer versions of these dashboards could look significantly different from earlier versions and add or remove certain metrics.
+
+This folder contains a dashboard that can be used to monitor hosts based on metrics ingested via OpenTelemetry collectors using the `hostmetrics` receiver and `resourcedetection` processor. The dashboard is in JSON format and can be uploaded to your Dynatrace tenant by [following the steps in the Dynatrace documentation](https://docs.dynatrace.com/docs/shortlink/dashboards-use#dashboards-upload).
+
+![A screenshot of the host dashboard providing an overview of used system resources](img/host-dashboard_1.png)
+
+## Prerequisites
+
+Dynatrace accepts metrics data with delta temporality via OTLP/HTTP.
+Collector and Collector Contrib versions v0.107.0 and above as well as Dynatrace Collector versions v0.12.0 and above support exporting metrics data in that format.
+Earlier versions ignore the `temporality_preference` flag and would, therefore, require additional processing (cumulative to delta conversion) before ingestion.
```

Comment:
i think we can actually remove this part, as the config below does not make use of the `temporality_preference` option